app/concepts/matestack/ui/core/collection/content/previous/previous.haml

```diff
@@ -0,0 +1,3 @@
+%a{@tag_attributes, "@click": "previous()"}
```

Comment:
Use a button maybe?

Reply:
because not everybody would like to use a button. using an `a` tag gives the matestack user more flexibility